### ISSUE TITLE
Openshift plugin being executed on each item in namespace caused timeout

### ISSUE THREAD

**Issue by phuongatemc** [konveyor/openshift-velero-plugin#115]
When there are many thousands of custom resources in the target namespace, OpenShift-velero plugin would be executed on each item.  This causes the total time of Velero backup increase significantly which is more than 10 minutes for around 5000-1000 custom resources in the namespace.  Compare to back up similar namespace in normal Kubernetes cluster (without OpenShift Velero plugin) which only takes a few seconds.

OpenShift Velero plugin is necessary for many resources such as Pod etc. but it should not have no effect on custom resources and therefore must not be executed for them.

**Comment by phuongatemc** [comment]
We still need this.

**Comment by kaovilai** [comment]
Hey @phuongatemc can you check if https://github.com/openshift/openshift-velero-plugin/pull/133 resolves this issue?

**Comment by phuongatemc** [comment]
@kaovilai  Could you elaborate more on how to use #133 ?  Is there any setting I need to do or it should work with latest OADP as default behavior?

**Comment by sseago** [comment]
@phuongatemc There's no configuration needed. This essentially refactors the plugin code to
1) do some things less often (actions needed only for imagestreams aren't done in the common plugin anymore)
2) cache some data in the plugin golang struct so that it only has to be looked up once rather than for every resource.

**Comment by kaovilai** [comment]
This behavior will be enabled in release 1.0.3 but there's a slight regression for multiple BSL.
https://github.com/openshift/openshift-velero-plugin/pull/144 should be included in 1.0.4.

You can try latest by plugin change by specifying image override like so.
```yaml
apiVersion: oadp.openshift.io/v1alpha1
kind: DataProtectionApplication
metadata:
  name: dpa-sample
spec:
  unsupportedOverrides:
    openshiftPluginImageFqin: quay.io/konveyor/openshift-velero-plugin:latest
```

Verify your override has applied by checking for `quay.io/konveyor/openshift-velero-plugin:latest` when running this command.
```sh
oc get deployment velero -n openshift-adp -ojsonpath="{.spec.template.spec.initContainers[?(@.name=='openshift-velero-plugin')].image}"
```
